### PR TITLE
Personal count as total tree count for TPOs

### PR DESCRIPTION
**Pull request by rishsingh07**
Personal count as total tree count for TPOs

### CODE REVIEW

File: src/features/user/UserProfile/components/UserInfo.tsx [hunk 1]
```diff
@@ -13,15 +13,23 @@ export default function UserInfo({
...
+      handleAddTargetModalOpen={handleAddTargetModalOpen}
+      authenticatedType={authenticatedType}
+      target={userprofile.score.target}
+      planted={userprofile.score.personal}
+    /> : 
+    <TreeCounter
         handleAddTargetModalOpen={handleAddTargetModalOpen}
         authenticatedType={authenticatedType}
         target={userprofile.score.target}
         planted={userprofile.score.personal + userprofile.score.received}
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/UserInfo.tsx:30]: Why didn't you add the if clause `{userprofile.type == "tpo" ? userprofile.score.personal : userprofile.score.personal + userprofile.score.received}` here if only this line changed? (I have not yet tested this.)

Comment (rishsingh07) [reply src/features/user/UserProfile/components/UserInfo.tsx]: Done

**Review by norbertschuler — COMMENTED** [review]
The code works, but as I do not know why we are doing this change, I cannot give approval. @sagararyal have you requested this?

**Comment by rishsingh07** [comment]
> The code works, but as I do not know why we are doing this change, I cannot give approval. @sagararyal have you requested this?

yes

**Review by sagararyal — APPROVED** [review]

**Pull request merged**